src/main/kotlin/rhmodding/bread/editor/AnimationsTab.kt

```diff
@@ -170,6 +174,62 @@ open class AnimationsTab<F : IDataModel>(editor: Editor<F>) : EditorSubTab<F>(ed
                     }
                 }
             }
+            children += HBox().apply {
+                styleClass += "hbox"
+                alignment = Pos.CENTER_LEFT
+                children += Button("Export Step").apply {
+                    tooltip = Tooltip("Exports current frame of animation to a PNG file.")
```

Comment:
"Exports the current frame of the animation to a PNG file. The current panning and zoom level will be used."